config/initializers/rack_attack.rb

```diff
@@ -154,6 +154,19 @@ def path_without_extentions
     match_data[:account_id] if match_data.present?
   end
 
+  # Throttle by individual user (based on uid)
+  throttle('/api/v2/accounts/:account_id/reports/user', limit: ENV.fetch('RATE_LIMIT_REPORTS_API_USER_LEVEL', '100').to_i, period: 1.minute) do |req|
```

Comment:
@vishnu-narayanan  these requests should be originating using the api_access_token . so lets limit it based on that.    think we can leave out the case where its regular users.

Reply:
True. Will make the change.